src/python/pants/engine/internals/scheduler_test.py

```diff
@@ -139,7 +139,7 @@ async def a_typecheck_fail_test(wrapper: TypeCheckFailWrapper) -> A:
 async def c_unhashable(_: TypeCheckFailWrapper) -> C:
     # This `await` would use the `nested_raise` rule, but it won't get to the point of raising since
     # the hashability check will fail.
-    _ = await Get(A, B, list())  # noqa: F841
+    _result = await Get(A, B, list())  # noqa: F841
```

Comment:
The two underscores were not of the same type, which mypy now catches, thus the new name here.